core/renderers/block_rendering_rewrite/block_render_draw_debug.js

```diff
@@ -186,20 +199,20 @@ Blockly.blockRendering.Debug.prototype.drawRenderedRow = function(row, cursorY)
  * Draw debug rectangles for a non-empty row and all of its subcomponents.
  * @param {!Blockly.BlockSvg.Row} row The non-empty row to render.
  * @param {number} cursorY The y position of the top of the row.
+ * @param {!Blockly.blockRendering.RenderInfo} info Rendering information about
+ *     the block to debug.
  * @package
  */
-Blockly.blockRendering.Debug.prototype.drawRowWithElements = function(row, cursorY) {
-  var cursorX = 0;
+Blockly.blockRendering.Debug.prototype.drawRowWithElements = function(row, cursorY, info) {
```

Comment:
It looks like you can just pass in RTL instead of info to all of these child functions.

Reply:
Done